### PR TITLE
Simplify driver and store call in tests

### PR DESCRIPTION
**Pull request by LauraSManns**

### CODE REVIEW

**Review by JensGM — CHANGES_REQUESTED** [review]
I think this pattern is used in many places and should be put in helpers in a function or something like that

File: xun/tests/helpers.py [hunk 1]
```diff
@@ -121,6 +121,13 @@ def __exit__(self, exc_type, exc_val, exc_tb):
         return exc_type is None
 
 
+def run_in_process(blueprint):
+    return blueprint.run(
+        driver=xun.functions.driver.Sequential(),
+        store=xun.functions.store.Memory()
+    )
```
Comment (mrSundvoll) [inline xun/tests/helpers.py:128]: I don't quite understand the function name

**Review by mrSundvoll — APPROVED** [review]
Looks good to me! :+1:

**Review by mrSundvoll — CHANGES_REQUESTED** [review]
Looks good, but we have to rebase first

**Review by mrSundvoll — APPROVED** [review]

**Review by JensGM — CHANGES_REQUESTED** [review]
You are missing 8 instances. Otherwise good!

There is also a formatting error pointed out by CircleCI. Have you checked it?

File: xun/tests/helpers.py (hunk 1 above)
Comment (JensGM) [reply xun/tests/helpers.py:130]: It's a sentance, you have an action (run_in_process), that you apply to "blueprint"

**Comment by LauraSManns** [comment]
Yes, the formatting error is within the with ... statement.
I added the eight forgotten instances.

**Review by JensGM — APPROVED** [review]

File: xun/tests/helpers.py (hunk 1 above)
Comment (mrSundvoll) [reply xun/tests/helpers.py:130]: Okey, fair enough

**Review by mrSundvoll — APPROVED** [review]

**Pull request merged**